### ISSUE TITLE
[Heroic] Culling of Stratholme Mal'Ganis Chest Bugged

### ISSUE THREAD

**Issue by arestavo** [dalaranwow/dalaran-wow#5734]
![Untitled](https://user-images.githubusercontent.com/50332138/59167937-d2818b00-8af0-11e9-840a-64e2989dc936.png)
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote 

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
This is the second time that Mal'Ganis' chest rolled loot, but didn't give it out because someone left before it was distributed. Also, I didn't get a badge of Triumph from it, although 2 party members got badges of heroism. A third run confirmed this.
**Expected behaviour**: Tell us what should happen instead
Loot from this chest should be awarded after rolls go out, and the chest should drop an appropriate badge.
**Steps to reproduce the problem**:

1. Defeat Mal'Ganis in Heroic Culling of Stratholme.
2. Loot chest.
3. Loot gets rolled, but not distributed if someone leaves before it's given out, and no badge of Triumph given even if no one leaves party early.

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**
http://i.imgur.com/RFouVVi.png
![Untitled2](https://user-images.githubusercontent.com/50332138/59168005-18d6ea00-8af1-11e9-8eee-246f5631854f.png)

**Comment by elliotmendiola** [comment]
I too received no badges from completion

![WoWScrnShot_061019_114025](https://user-images.githubusercontent.com/16568775/59207366-ed95de80-8b74-11e9-8f86-d8477c4be56b.jpg)

**Comment by Lothloryen** [comment]
Hello there,

Thank you very much for the report, I will look into it through #5257
